soh/include/z64save.h

```diff
@@ -265,6 +265,7 @@ typedef struct {
     /*        */ char childAltarText[250];
     /*        */ char adultAltarText[750];
     /*        */ char ganonHintText[150];
+    /*        */ char gregHintText[250];
```

Comment:
just out of curiosity what drives these lengths? they look arbitrary

Reply:
they are kind of arbitrary, might be worth going through and figuring out the exact max lengths based on different locations etc. at some point but that just hasn't happened yet